resources/views/store/product.blade.php

```diff
@@ -131,13 +131,39 @@ class="js-gallery"
             @endif
         </div>
 
-        @if($product->inStock())
         <div class="osu-layout__sub-row osu-layout__sub-row--with-separator" id="add-to-cart">
             <div class="big-button">
-                <button type="submit" class="js-store-add-to-cart btn-osu btn-osu-default">Add to Cart</button>
+                <button type="submit" class="js-store-add-to-cart btn-osu btn-osu-default"
+                    @if(!$product->inStock())
+                        style="display: none;"
+                    @endif
+                >
+                {{ trans('store.product.add-to-cart') }}</button>
+            </div>
+
+            <div class="big-button">
+                <button type="submit" class="js-store-notify btn-osu btn-osu-default" formaction="{{ route('store.request-notification', ['product_id' => $product->product_id]) }}" data-remote="true"
```

Comment:
then i would have to change the request notification method from post to get, what wouldn't make quite sense provided that we actually change the data on server